src/main/java/frc/robot/subsystems/TurretSubsystem.java

```diff
@@ -134,6 +142,92 @@ public Command stowTurret() {
     return new InstantCommand(() -> setTurretPos(Math.toRadians(0))).withName("StowTurretCommand");
   }
 
+  public Command zeroTurretCommand() {
+    return Commands.runOnce(() -> this.zeroTurret(), this).withName("ZeroTurretCommand");
+  }
+
+  /** Will calculate the current turret position and update encoders and motors off of it. */
+  public void zeroTurret() {
+    double lowGearPosition = lowGearCANcoder.getAbsolutePosition().getValue().doubleValue();
+    double highGearPosition = highGearCANcoder.getAbsolutePosition().getValue().doubleValue();
```

Comment:
The CANCoderConfiguration will be important to setup and be sure of. Otherwise the absolute positions may not correlate between the two sensors and the turret position.